graphql/mappers/contact-employment-insurance.js

```diff
@@ -0,0 +1,100 @@
+import clientQuery from '../client'
```

Comment:
I also think it'd be beneficial to integrate this mapper into the contact-employment-insurance page in this PR, that way we can tell if the GQL query and mapper are working the way we want them to before its merged. Should be as simple as importing this file and substituting the parseContactMethods function